### ISSUE TITLE
Spaces added before tables in questions

### ISSUE THREAD

**Issue by cedrick-f** [QuizandSurveyMaster/quiz_master_next#916]
Hi,

##Site Info

WordPress Version: 4.9.13
QSM Version: 6.4.5
Browser: Firefox

##General description
If a `<table>` tag is used in question : 
```
Your new question!
<table>
<tbody>
<tr>
<td>1</td>
<td>2</td>
</tr>
<tr>
<td>3</td>
<td>4</td>
</tr>
</tbody>
</table>
```
![image](https://user-images.githubusercontent.com/11462584/81163468-198d3000-8f8f-11ea-8c18-821643a83c46.png)

![image](https://user-images.githubusercontent.com/11462584/81163641-5e18cb80-8f8f-11ea-9c6f-d4bb2b03e8b9.png)

then it appears preceded by several carriage returns 
in preview
![image](https://user-images.githubusercontent.com/11462584/81163732-7c7ec700-8f8f-11ea-8797-77ee52200c0d.png)

after a preview
![image](https://user-images.githubusercontent.com/11462584/81163816-9ddfb300-8f8f-11ea-97d7-0324d16ab0c6.png)

Thanks by advance
Cédrick

**Comment by usman297** [comment]
Hello,
Thanks for your query. For all your support queries, please raise a ticket at https://support.quizandsurveymaster.com/ and let me know. Our support team will get back to you shortly.

**Comment by etvarun** [comment]
Hi @cedrick-f,

We have fixed this bug in the latest release (6.4.11)

Regards,
Varun
